lib/world.rb

```diff
@@ -50,6 +50,20 @@ def debug_in_after_hook
       end
     end
 
+    def print_debugging_info(host)
+      # Intention of method is to execute additional debug statement to analyze unexpected trickier failures.
+      logger.handled_error = ''
+      res = admin.cli_exec(:get, resource: "nodes", _quiet: true)
+      nodes = host.exec_raw("echo '#{res[:response]}' | grep -E 'SchedulingDisabled|NotReady' | awk '(NR>1)'", quiet: true)
+      res = admin.cli_exec(:get, all_namespaces: true, resource: "co", _quiet: true)
```

Comment:
ClusterOperator is not a namespace scope resource. `all_namespaces` is not needed here.

Reply:
Updated. Thank you.